File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -447,6 +520,85 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-and-highlight-in-PowerPoint).
 
+## Find and highlight using Regex pattern 
+
+You can find either the first occurrence or all the occurrences of a pattern of text in a PowerPoint presentation using the `Find` or `FindAll` method and highlight it.
+
+The following code example illustrates how to find all the occurrences of a pattern of text using Regex pattern and highlight it.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Load or open an PowerPoint Presentation.
+using (FileStream inputStream = new FileStream("Sample.pptx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    //Open an existing PowerPoint presentation.
+    using (IPresentation pptxDoc = Presentation.Open(inputStream))
+    {
+        //Find all the occurrences of given pattern of text in the PowerPoint presentation using Regex.
```

Comment:
//Find all the occurrences of a given pattern of text in the PowerPoint presentation using Regex.